packages/bitget-wallet/src/lib/bitget-wallet.ts

```diff
@@ -0,0 +1,284 @@
+import { isMobile } from "is-mobile";
+import type {
+  WalletModuleFactory,
+  WalletBehaviourFactory,
+  InjectedWallet,
+  Transaction,
+  Optional,
+  Account,
+} from "@near-wallet-selector/core";
+import { waitFor } from "@near-wallet-selector/core";
+import type { InjectedBitgetWallet } from "./injected-bitget-wallet";
+import icon from "./icon";
+
+declare global {
+  interface Window {
+    bitkeep: {
+      near: InjectedBitgetWallet | undefined;
+    };
+  }
+}
+
+export interface BitgetWalletParams {
+  iconUrl?: string;
+  deprecated?: boolean;
+}
+
+interface BitgetWalletState {
+  wallet: InjectedBitgetWallet;
+}
+
+const isInstalled = () => {
+  return waitFor(() => !!window.bitkeep.near).catch(() => false);
```

Comment:
Is `window.bitkeep.near` injected in the current version of Bitget Wallet extension, or is it still not live yet? 

When checking on console it returns undefined:
![image](https://github.com/near/wallet-selector/assets/95851345/e50aee65-b981-47f4-aa2c-2e9c034defbd)

Even though I have installed Bitget Wallet extension we still see this view since `window.bitkeep.near` is undefined:

![image](https://github.com/near/wallet-selector/assets/95851345/fd8b3f8e-183b-488c-bf80-4810bb7f693b)